### DISCUSSION TITLE
Can vision transformer be deployed on the arm architecture?

### DISCUSSION THREAD

**Discussion by ywfwyht** [open-mmlab/mmdeploy#2074] [General]
Hi. Can vision transformer be deployed on the arm architecture?

**Comment by irexyc** [comment]
For arm architecture, you cau build mmdeploy with ncnn as backend.  And ncnn supports vit model

**Reply by ywfwyht** [reply]
> For arm architecture, you cau build mmdeploy with ncnn as backend. And ncnn supports vit model

thanks
